### PR TITLE
15538 Fixed validation for relationship types

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#15538

*Description of changes:*
- app version = 5.11.19
- imported latest Relationships Panel shared component
- adjusted whitespace
- passed in validation prop

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-732-wdo4f2w7.web.app

File: src/components/Restoration/RestorationType.vue
Comment (severinbeauvais) [inline src/components/Restoration/RestorationType.vue:1]: Before:
image

After:
image

Comment (severinbeauvais) [reply src/components/Restoration/RestorationType.vue:1]: After, with validation enabled (ie, have visited review page):
image

**Review by JazzarKarim — APPROVED** [review]

**Review by ArwenQin — APPROVED** [review]

File: src/components/Restoration/RestorationType.vue
Comment (ArwenQin) [reply src/components/Restoration/RestorationType.vue:1]: Looks good!

**Pull request merged**